src/coreclr/jit/compiler.h

```diff
@@ -9432,7 +9432,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 // likely complicated enough that loop alignment will not impact performance.
 #define DEFAULT_MAX_LOOPSIZE_FOR_ALIGN DEFAULT_ALIGN_LOOP_BOUNDARY * 3
 
-// By default only single iteration loops will be unrolled
+// By default only loops with a constant limit less than or equal to this will be unrolled
```

Comment:
The iteration count is not the same as the limit.
```suggestion
// By default only loops with a constant iteration count less than or equal to this will be unrolled
```